qaes/encryption.py

```diff
@@ -34,6 +34,11 @@ def decrypt(self, cipher_text: str) -> str:
     def generate_iv(self):
         return secrets.token_bytes(self.block_size)
 
+    def constant_iv(self, plain: bytes):
+        iv = plain * (len(plain) // self.block_size + 2)
+        iv = iv[:self.block_size]
+        return iv
```

Comment:
This code doesn't provide a block. I double checked on `aes.encrypt('a', randomly=False)`. It always raise `ValueError: Invalid IV size` exception.

Reply:
You have to make sure that the IV is exaclty a block; No more, no less.

Reply:
Fix.